gstd/src/msg/basic.rs

```diff
@@ -216,6 +216,43 @@ pub fn reply_bytes<T: AsRef<[u8]>>(payload: T, value: u128) -> Result<MessageId>
     gcore::msg::reply(payload.as_ref(), value).into_contract_result()
 }
 
+/// Send a new message as [`reply`] function but with concrete gas limit.
```

Comment:
Didn't you prefer to place here something like "same as `reply(..)`, but with explicit gas limit" ?